zebra-network/src/protocol/external/codec.rs

```diff
@@ -590,20 +599,38 @@ impl Codec {
         Ok(Message::Mempool)
     }
 
-    fn read_filterload<R: Read>(&self, mut _reader: R) -> Result<Message, Error> {
-        return Err(Error::Parse("filterload messages are not implemented"));
-    }
+    fn read_filterload(&self, mut reader: Cursor<&BytesMut>) -> Result<Message, Error> {
```

Comment:
This might be more futureproof if it kept the generic reader but passed the `body_len` from the caller's scope:
```suggestion
    fn read_filterload<R: Read>(&self, mut reader: R, body_len: usize) -> Result<Message, Error> {
```
so that `body_len` doesn't have to be recomputed and the reader details don't leak into the filterload function.